ffi/src/scan.rs

```diff
@@ -273,6 +273,20 @@ pub struct CStringMap {
     values: HashMap<String, String>,
 }
 
+impl CStringMap {
```

Comment:
should this be
```suggestion
impl Default for CStringMap {
```
(surprised clippy didn't flag it -- usually it complains if a struct defines methods from standard traits, without the actual trait impl)

Reply:
Actually... should be able to just `#[derive(Default)]`?